CMakeLists.txt

```diff
@@ -50,14 +50,16 @@ endif()
 option(BUILD_TORCHTEXT_PYTHON_EXTENSION "Build Python extension" OFF)
 
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+set(TORCH_INSTALL_PREFIX "${CMAKE_PREFIX_PATH}/../.." CACHE STRING "Install path for torch")
```

Comment:
This seems to be overwriting the value provided by Python process. Is this correct? I think ‘option’ is more suited here.

Reply:
According to the [docs](https://cmake.org/cmake/help/latest/command/set.html#set-cache-entry), "Since cache entries are meant to provide user-settable values this does not overwrite existing cache entries by default. Use the FORCE option to overwrite existing entries.".

IIUC, `option` is only used for booleans ([docs](https://cmake.org/cmake/help/latest/command/option.html)).

Reply:
Ah I see. I mistook them.